src/GitLink/GitLink.csproj

```diff
@@ -13,14 +12,12 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
-    <NuGetPackageImportStamp>
-    </NuGetPackageImportStamp>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\output\debug\GitLink\</OutputPath>
+    <OutputPath>bin\debug\</OutputPath>
```

Comment:
Why change this? It should be irrelevant for msbuild where the files are being written to, and this way we can simply delete the /output to clear the output without having to open vs or move into subdirectories.